### ISSUE TITLE
🌟 [EPIC] Brainstorming von IT4C-Woche

### ISSUE THREAD

**Issue by mahula** [IT4Change/IT4C.dev#59]
### Gradido
  - Deployment angehen in Absprache mit Bernd (Matthias wieder kontaktieren), sollte jetzt bereits mit aktuellem Repo-Stand machbar sein
  - auch Branding beim Deployment anbieten?
  - Code Hash (@ogerly) für Kernkomponenten der Software zur Absicherung von Kernfunktionen und Konfigurationen in Bezug auf x-Community SendCoin
  - Gradido-Rat
  - Sicherheitsaspekte der Software
  
#### Struktur der Internetseiten sortieren
- Repo-Strukturaufzeichnung vom Treffen (@ogerly, Emil) hier hinterlegen
- Produktbeschreibung (mit Buchungsoption z.B. One-Click Deployment) auf IT4C Website
- auf `ocelot.social` Projektbeschreibung statt der Readmes
- Subdomain `docs.ocelot.social` für die reine eher technische Dokumentation (mittels GitBook aus Readmes und Summaries Buch erstellen)

#### Roadmap - nächste Features
Was sind sinnvolle und nützliche Feature für eine weitere Entwicklung von Ocelot?
Dafür sollten wir eine Roadmap erstellen
- der Newsfeed
- https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/6493
- https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/6541
- https://github.com/Ocelot-Social-Community/Ocelot-Social/issues/6492

#### Einrichtungsaufwand
- Aufschlüsselung der Arbeitsschritte und Kosten für eine komplette Ersteinrichtung und Betriebskosten des Servers hinterlegen (z.B. bei Produktbeschreibung) (siehe Emils Aufzeichnungen mithinterlegen) --> das einfließen lassen in allgemeinen IT4C Produkte- und Leistungskatalog
- Anteil erbrachter Leistung für/in den Gemeinschaftstopf

#### Branding
  - wie vorhandene Branding-Funktionalität verbinden mit Option für Nutzer beim Einrichten Farben etc. zu ändern
  - vorerst möglichst einfache Methode schaffen/anwenden, um Branding anpassen zu können

#### Projektkommunikation
- Überlegung, Issues bzw. Repos für interne Issues privat zu halten
- reicht uns für Internes Discord weiterhin aus?

#### Team(s)struktur
- Rechtsform(enstruktur)
- Busfaktor
- ein Team, Teamverbund, Kollektiv?

#### Automatisiertes Deployment
- Vereinfachung und Automatisierung verfeinern (allgemein für alle Projekte der IT4C Gruppe) @Elim @ulfgebhardt 
  - bestehende Projekte (self-hosted) One-Click Installation (Tech Stack, Vorgehen)
  - Deloyment selbst auf dem Server
    - ausreichend graphisches Feedback und Eingabemöglichkeiten beim Einrichten
    - hier eher weniger und default Einstellungen
  - Netzwerkadmin soll angemessene Möglichkeiten haben, seine gekaufte Instanz zu konfigurieren (z.B. Logo, Theme, etc.) 
  - Neben Kubernetes weitere Deployment-Variante (z.B. Docker-Netzwerk) anbieten?
  - Recherche, wie solche Prozesse bei Hosting-Anbietern graphisch gestaltet sind,was unserem Flow dienlich wäre

#### Ocelot und Gradido
- Anwendung des Netzwerks verbreitern der Betreiberbasis (mehr Betreiber und mehr Nutzer)
- die Anwendung in Fahrt bringen

#### Produkt- und Leistungskatalog
- Überlegungen Gesamtstruktur (IT4C, Busfaktor, InfinityLabs, Ocelot, Gradido, etc.) (siehe Emils Aufzeichnungen mithinterlegen)
- Repräsentanz fertigstellen (Website)
- Manifest?
- Repräsentation des Team, um auch unabhängig der aktuellen Projekte sichtbar für Arbeitsangebote zu sein
- mit InfinityLabs das gemeinsam angehen (siehe auch Emils Vorschlag (externe) Schnittstelle zw. Anfragen und uns einrichten)

#### Rollen
- Delpoyment
- Development
- Administratives (PO, Scrum Master, Kommunikation in alle Richtungen, Rechnungswesen)
- Support (Kette an Kommunikation)?
- PR (Außenkommunikation, Werbung, etc.)

#### Support
- kurze Wege für Abläufe Support-Anfragen - Priorisierung - Kommunikation - Lösung/Rückmeldung

#### Funding
- für was genau?
- welche Größenordnung?
- zu welchem Preis, welchen Abhängigkeiten?
- wer kümmert sich darum?

**Comment by ulfgebhardt** [comment]
I don't think this is still relevant. If I am wrong, please reopen and clarify what this issue is about
